src/util/replace_symbol.h

```diff
@@ -82,6 +83,17 @@ class replace_symbolt
   replace_symbolt();
   virtual ~replace_symbolt();
 
+  const expr_mapt &get_expr_map() const
```

Comment:
Couldn't you avoid exposing these by providing an `optionalt find(x)` method?

Reply:
That wouldn’t be sufficient, but it would of course be possible to move a lot of code from the constant propagation in here. Let me know if that’s the preferred route!